doc/helpers/Input-extensions.md

```diff
@@ -0,0 +1,27 @@
+# InputExtensions Attached Properties
+Provides various attached properties for _input controls_, such as `TextBox` and `PasswordBox`.
+
+## Properties
+Property|Type|Description
+-|-|-
+AutoDismiss|bool|Whether the soft-keyboard will be dismissed when the enter key is pressed.
```

Comment:
Personal preference: include Keyboard in the name, because that is what is dismissed.

Reply:
> We can actually just call it AutoDismiss, as we're in the context of InputExtensions already 
> \- FT

Reply:
Input and keyboard are 2 different things :D

Reply:
We are keeping the existing property name _for now_.